here/fleet_telematics.go

```diff
@@ -0,0 +1,129 @@
+			EstimatedArrival     time.Time `json:"estimatedArrival"`
+			EstimatedDeparture   time.Time `json:"estimatedDeparture"`
+			FulFilledConstraints []string  `json:"fulfilledConstraints"`
+		} `json:"waypoints"`
+		Distance         string `json:"distance"`
+		Time             string `json:"time"`
+		InterConnections []struct {
+			FromWaypoint string  `json:"fromWaypoint"`
+			ToWaypoint   string  `json:"toWaypoint"`
+			Distance     float64 `json:"distance"`
+			Time         float64 `json:"time"`
+			Rest         float64 `json:"rest"`
+			Waiting      float64 `json:"waiting"`
+		} `json:"interconnections"`
+		Description   string `json:"description"`
+		TimeBreakdown struct {
+			Driving int `json:"driving"`
+			Service int `json:"service"`
+			Rest    int `json:"rest"`
+			Waiting int `json:"waiting"`
+		}
+	}
+	Errors             []string `json:"errors"`
+	ProcessingTimeDesc string   `json:"processingTimeDesc"`
+	ResponseCode       string   `json:"responseCode"`
+}
+
+// newFleetTelematicsService returns a new RoutingService.
+func newFleetTelematicsService(sling *sling.Sling) *FleetTelematicsService {
+	return &FleetTelematicsService{
+		sling: sling,
+	}
+}
+
+// Returns destinations as a formatted string.
+func createDestination(destination DestinationParams) string {
+	if destination.Text != "" {
+		return fmt.Sprintf("%s;%f,%f;", destination.Text, destination.Coordinates[0], destination.Coordinates[1])
+	}
+	return fmt.Sprintf("%f,%f;", destination.Coordinates[0], destination.Coordinates[1])
+}
+
+// CreateFleetTelematicsParams creates fleet telematics parameters struct.
+func (s *FleetTelematicsService) CreateFleetTelematicsParams(start DestinationParams, end DestinationParams, destinations []DestinationParams, apiKey string, modes []Enum) FleetTelematicsParams {
+	var buffer bytes.Buffer
+	for _, routeMode := range modes {
+		mode := Enum.ValueOfRouteMode(routeMode)
+		buffer.WriteString(mode + ";")
+	}
+	routeModes := buffer.String()
+	routeModes = routeModes[:len(routeModes)-1]
+	fleetTelematicsParams := FleetTelematicsParams{
+		Start:     createDestination(start),
+		APIKey:    apiKey,
+		Modes:     routeModes,
+		Departure: "now",
+		End:       createDestination(end),
+	}
+
+	for i, destination := range destinations {
+		stringDestination := createDestination(destination)
+		concatenated := "Destination" + strconv.Itoa(i+1)
+		reflect.ValueOf(&fleetTelematicsParams).Elem().FieldByName(concatenated).SetString(stringDestination)
+	}
+
+	fmt.Printf("%v", fleetTelematicsParams)
+	return fleetTelematicsParams
+}
+
+// FindSequence with given parameters.
+func (s *FleetTelematicsService) FindSequence(params *FleetTelematicsParams) (*FleetTelematicsWaypointsSequenceResponse, *http.Response, error) {
+	routeSequence := new(FleetTelematicsWaypointsSequenceResponse)
+	apiError := new(APIError)
+	resp, err := s.sling.New().Get("findsequence.json").QueryStruct(params).Receive(routeSequence, apiError)
+	fmt.Printf("\n%v\n", routeSequence)
```

Comment:
Please remove lines that prints sequence and response.

Reply:
Removed. Sorry, I was using it for debugging and forgot to remove it.